asset_stream_manager/main.cc

```diff
@@ -160,21 +167,24 @@ ABSL_FLAG(std::string, dev_mount_dir, "",
 int main(int argc, char* argv[]) {
   absl::ParseCommandLine(argc, argv);
 
-  // Set up config. Allow overriding this config with
-  // %config_dir%\assets_stream_manager.json.
-  const std::string config_path = cdc_ft::path::Join(
-      absl::GetFlag(FLAGS_config_dir), "assets_stream_manager.json");
+  // Set up config. Allow overriding this config |config_file|.
   cdc_ft::AssetStreamConfig cfg;
-  absl::Status cfg_load_status = cfg.LoadFromFile(config_path);
+  std::string config_file = absl::GetFlag(FLAGS_config_file);
+  absl::Status cfg_load_status =
+      cdc_ft::path::ExpandPathVariables(&config_file);
+  if (cfg_load_status.ok()) {
+    cfg_load_status = cfg.LoadFromFile(config_file);
```

Comment:
Nit: I think you can simply use cfg_load_status.update(cfg.LoadFromFile(config_file));

Reply:
Done